lifecycle-operator/chart/templates/deployment.yaml

```diff
@@ -99,6 +99,9 @@ spec:
         - name: OPTIONS_CONTROLLER_LOG_LEVEL
           value: {{ .Values.lifecycleOperator.env.optionsControllerLogLevel | quote
             }}
+        - name: TTL_SECONDS_AFTER_FINISHED
+          value: {{ .Values.lifecycleOperator.env.ttlSecondsAfterFinished | quote
+            }}
```

Comment:
we don't need this to be an env var. We can hardcode a default in code directly since users can change the value directly via CRDs.